client/app/bundles/comments/components/Navigationbar/Navigationbar.jsx

```diff
@@ -0,0 +1,77 @@
+import React, {PropTypes} from 'react';
```

Comment:
This is our new navigation bar written in React/Redux. It used a shared redux store so it will update when the shared store changes, such as when a comment is added.